docs/tutorials/autoscaling-federatedhpa-with-cronfederatedhpa.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Autoscaling FederatedHPA with CronFederatedHPA
+---
+
+In Karmada, a CronFederatedHPA scales workload's replicas (any kinds of workload with scale subresource, e.g. Deployment) or FederatedHPA's minReplicas/maxReplicas, with the aim of scaling the business in advance to meet a sharp load peak.
+
+If autoscaling deployment directly, the deployment may lose the elasticity and could not meet the business demand at real time.
+
+So we suggest to use CronFederatedHPA to scale the FederatedHPA, which manages the elasticity of the deployment at the same time. You can scale the workloads up in advance, also FederatedHPA can scale the workloads up more if the business demand is too high.
+
+This document walk you through an example of enabling CronFederatedHPA to a FederatedHPA.
+
+## Prerequisites
+
+### Karmada has been installed
+
+We can install Karmada by referring to [Quick Start](https://github.com/karmada-io/karmada#quick-start), or directly run `hack/local-up-karmada.sh` script which is also used to run our E2E cases.
+
+## Deploy workload in `member1` and `member2` cluster
+We need to deploy deployment(1 replica) in member1 and member2.
```

Comment:
It might be confusing if you want to deploy `1 replica` on `2` clusters.
Maybe we need more description here.

Reply:
or can we start with `2` replicas?